### PR TITLE
Added "feels".

### PR DESCRIPTION
**Pull request by el97**
Small changes in the sv lang...

### CODE REVIEW

File: translations/sv.json [hunk 1]
```diff
@@ -29,4 +29,5 @@
 	"UPDATE_NOTIFICATION": "MagicMirror² uppdatering finns tillgänglig.",
 	"UPDATE_NOTIFICATION_MODULE": "Uppdatering finns tillgänglig av {MODULE_NAME} modulen.",
 	"UPDATE_INFO": "Denna installation ligger {COMMIT_COUNT} steg bakom {BRANCH_NAME} grenen."
```
Comment (atiris) [inline translations/sv.json]: Missing comma at the end of the row

**Comment by MichMich** [comment]
hi @el97 as @atiris states there is an missing comma in your translation change. If you fix this and add an updated CHANGELOG I can merge your PR.

Thanks,

**Comment by MichMich** [comment]
Please send a new PR to the develop branch.

**Comment by fewieden** [comment]
you need to add it manually to the changelog file

**Pull request closed without merging**